### PR TITLE
Replace h1 with new logo

### PR DESCRIPTION
**Pull request by koresar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-# Stampit
-[![Travis-CI](https://travis-ci.org/stampit-org/stampit.svg)](https://travis-ci.org/stampit-org/stampit)
-[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/stampit-org/stampit?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+<img src="https://raw.githubusercontent.com/stampit-org/stampit-logo/master/stampit-logo.png" width="320" />[![Travis-CI](https://travis-ci.org/stampit-org/stampit.svg)](https://travis-ci.org/stampit-org/stampit)[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/stampit-org/stampit?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (ericelliott) [inline README.md]: Use the inline image format: `Stampit`:

Stampit

Comment (koresar) [inline README.md]: Are you sure you want the logo 100% wide and take extra vertical scrolling space?

Comment (ericelliott) [inline README.md]: No. =)

Comment (koresar) [inline README.md]: :)

**Pull request merged**